### DISCUSSION TITLE
Joining selection contents?

### DISCUSSION THREAD

**Discussion by inducer** [helix-editor/helix#9671] [Q&A] [answered]
I have text that looks like the following:
```
Useful text A
Junk
Junk
Useful text B
Junk
Junk
Useful text C
Junk
Junk
```

and I would like
```
Useful text A
Useful text B
Useful text C
```

I was able to get to the point of having multiple selections convering the "Useful text" bits, but then I got stuck. What do I do to get the content of the selections pasted together, say separated by newlines/commas/what have you? Is there an efficient way?

Thanks!

**Comment by MrWheatley** [comment, marked as answer]
I think you can use the `yank_joined` and `yank_joined_to_clipboard` commands for that.
